catalyst_voices/apps/voices/lib/app/view/app.dart

```diff
@@ -56,6 +56,9 @@ class _AppState extends State<App> {
       BlocProvider<WorkspaceBloc>(
         create: (context) => Dependencies.instance.get<WorkspaceBloc>(),
       ),
+      BlocProvider<ProposalEditorBloc>(
```

Comment:
Should we consider to rename it to `ProposalBuilderBloc`? I see it is quite often referred to as `ProposalBuilder`, `ProposalEditor` is much less common.

Reply:
done. Renamed to `ProposalBuilder`